views/js/testRunner/actionBarHook.js

```diff
@@ -88,13 +94,56 @@ define([
             }, function(e){
                 errorHandler.throw(_ns, 'the hook amd module cannot be found');
             });
-            
+
         }else{
             errorHandler.throw(_ns, 'invalid tool config format');
         }
 
     }
 
+    /**
+     * Append a dom element $button to a $container in a specific order
+     * The orders are provided by data-order attribute set to the $button
+     * 
+     * @param {JQuery} $container
+     * @param {JQuery} $button
+     */
+    function _appendInOrder($container, $button){
+        
+        var order = $button.data('order');
+        if(order){
```

Comment:
if order is 0, then you'll be in the else block and the button will be appended. Is it what you'd like to achieve ? Otherwise check if order is a number